client/components/app.jsx

```diff
@@ -19,13 +25,32 @@ export default class App extends React.Component {
       .finally(() => this.setState({ isLoading: false }));
   }
 
+  setView(name, params) {
+    this.setState({
+      message: this.state.message,
+      isLoading: this.state.isLoading,
```

Comment:
You can remove `message` and `isLoading` from all of your `App` state. It was just for testing purposes.